src/models/Delear.test.ts

```diff
@@ -0,0 +1,43 @@
+import { Dealer } from './Dealer'
+
+describe('dealer test', () => {
+  it('returns array containing 13 unique numbers', () => {
+    const dealer = new Dealer()
+    const uniqueCards = dealer.createUniqueCards()
+    expect(new Set(uniqueCards).size).toBe(13)
```

Comment:
set 쓰면 만약 uniqueCards에 중복되는 카드가 있었다고 해도 중복 제거되서 false negative하게 될 것 같네여,,

```suggestion
    expect(uniqueCards.length).toBe(13)
```

Reply:
오우 놓칠뻔 했네요 감사합니다 🙏🙏